src/AiidaDFTK.jl

```diff
@@ -101,7 +102,36 @@ Run a DFTK calculation from a json input file.
 Output is by default written to `stdout` and `stderr`.
 The list of generated output files is returned.
 """
-function run_json(filename::AbstractString; extra_output_files=String[])
+function run_json(filename::AbstractString; extra_output_files=String[], min_version=typemin(VersionNumber), max_version=typemax(VersionNumber))
```

Comment:
For the version bounds I would just use a single string argument, which is parsed into a version range same as `Pkg` does it (https://pkgdocs.julialang.org/v1/compatibility/). I suppose there is a way to invoke Pkg for this parsing.

Reply:
Apparently the way to do it is via `<package version> ∈ Pkg.Versions.semver_spec(<spec>)`. Looks like it requires Pkg internals, but I think that's acceptable.

Reply:
That does not look like an internal. Or why should it be ?

Reply:
Still not too happy about the two versions here :smile: